main.scss

```diff
@@ -26,12 +26,13 @@ $o-expander-is-silent: true !default;
 @mixin oExpanderToggle($root-class-name: o-expander) {
 	@include oColorsFor(expander-text, text);
 	font-weight: normal;
-	font-size: 12px;
+	font-size: 16px;
```

Comment:
No, it was just to set them at the same size as the elements, but I can turn them back to their own size so it can be differentiated easily